browser/src/control/Control.UIManager.js

```diff
@@ -711,6 +711,85 @@ L.Control.UIManager = L.Control.extend({
 		this.map.fire('hidebusy');
 	},
 
+	showHyperlinkWarn: function(url, action) {
```

Comment:
Better to have a generic showWarnDialog function where we start to use slowly to convert simple dialogs of that type, starting with Hyperlink warning dialog